logistics_planning_invoicing/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    'name': 'Logistics Planning Invoicing',
+    'summary': '''
+        Adds link with Invoicing
+    ''',
+    'author': 'Solvos',
+    'license': 'LGPL-3',
+    'version': '13.0.1.0.0',
+    'category': 'stock',
+    'website': 'https://github.com/solvosci/slv-stock',
+    'depends': [
+        'account',
+        'logistics_planning_base',
+    ],
+    'data': [
+        'views/account_move_views.xml',
+        'views/logistics_planning_views.xml',
```

Comment:
This file should be renamed IMO:
```suggestion
        'views/logistics_schedule_views.xml',
```